index.js

```
@@ -187,6 +187,9 @@ function use (extensions) {
     this.on('close', onfinalize)
     this.on('end', onfinalize)
     this.on('finish', this.finalize)
+    this.on('error', function (err) {
+      debug('stream error', err)
+    })
```

this will swallow uncaught errors when debug isn't enabled, right?